exercises/D-logical-operators/exercise2.js

```diff
@@ -3,10 +3,42 @@
   ---------------------------------
   This program calls some functions that are either missing or incomplete.
   Update the code so that you get the expected result.
+
+  let num = 10;
+
+function satisfiesRequirements(num) {
+  if (num > 3 && num < 10) {
+    return true;
+  }
+
+  return false;
+}
 */
 
-function isNegative() {}
+function isNegative(num) {
+  if (num < 0) {
+    return true         
+  } else {
+    return false
+  }
```

Comment:
As we talked this can be written like this:
```
if (num < 0) {
  return true;
}
return false;
```
or if you'd like more advanced you can just write it like this. Can you guess why this is true?
```
return num <0;
```